docs/guides/private-submodules.rst

```diff
@@ -58,15 +56,21 @@ to give read access to several repositories using only one SSH key.
 Azure DevOps
 ------------
 
-Azure DevOps doesn't have per-project keys.
-If you already :ref:`added the SSH key from the main project <guides/importing-private-repositories:azure devops>`,
-you don't need to do anything else.
+Azure DevOps has per-user keys,
+as long as the user has access to your main repository and all its submodules,
```

Comment:
:+1: I'd say break this up a bit more to prevent running on in the sentence too much. Also, assuming devops calls them "repositories" here -- not sure if that is accurate.

```suggestion
Azure DevOps does not have per-repository SSH keys, but keys can be added to a user instead.
As long as this user has access to your main repository and all its submodules,
```

Reply:
yes, they call them repositories.